### PR TITLE
Update setup and virtual environment documentation

### PR DESCRIPTION
**Pull request by tomnatt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,21 @@
 and systems administration tasks. It allows the user to run commands across a
 server farm.
 
-## Local usage
+## Local usage and environment setup
+
+The tool is intended to be configured and run on your local workstation/laptop. You should use `virtualenv` to create an isolated Python environment.
```
Comment (cbaines) [inline README.md]: This is rather strong for my liking. Could you either add a justification for saying "you should" use virtualenv, or say something like "The installation instructions have been tested on a XXX system:".

I don't use virtualenv, and get on just fine.

Comment (tomnatt) [inline README.md]: This is a safe option for people who want to get up and running with minimal fuss (or indeed python knowledge). As ever, if you're happy doing something else then feel free to do so.

Comment (cbaines) [inline README.md]: While I don't necessarily agree with your reasoning, I'm happy for this to be merged if that is added.

Comment (timblair) [inline README.md]: Using virtualenv is the generally accepted practice for all our Python work. Sure, there are people who like to do something different, but I'm happy to document the more commonly trodden path.

**Pull request merged**